recover-id-object/scripts/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG build_image=rust:1.70.0
+ARG base_image=debian:buster-slim
+FROM ${build_image} AS build
+
+WORKDIR /usr/app/recover-id-object
+
+COPY recover-id-object/Cargo.toml recover-id-object/Cargo.lock .
+COPY deps /usr/app/deps
+RUN mkdir src && echo 'fn main() { println!("Dummy!"); }' > ./src/main.rs
```

Comment:
Used for caching. Meaning we only want to rebuild deps given that they actually change, and not when source code changes

Reply:
I literally wrote why I am doing that just above. You can go in and see this: https://github.com/Concordium/concordium-misc-tools/actions/runs/7877700035

and then compare it with this:

https://github.com/Concordium/concordium-misc-tools/actions/runs/7877546244

Which is exactly the difference between not using caching and then using it as I am.

This here indeed fixes the layers, such that they are alike and that is indeed a nobrainer in terms of optimising on the release actions.